installer/templates/phx_single/config/prod.exs

```diff
@@ -25,13 +25,17 @@ config :logger, level: :info
 #       ...
 #       url: [host: "example.com", port: 443],
 #       https: [
+#         ip: {0, 0, 0, 0},
 #         port: 443,
 #         cipher_suite: :strong,
 #         keyfile: System.get_env("SOME_APP_SSL_KEY_PATH"),
 #         certfile: System.get_env("SOME_APP_SSL_CERT_PATH"),
 #         transport_options: [socket_opts: [:inet6]]
 #       ]
 #
+# If you intend to run Phoenix behind a reverse proxy, consider
+# changing `ip` to `{127, 0, 0, 1}` to prevent direct access.
+#
```

Comment:
FYI this is correct only if phoenix and the reverse proxy is on the same machine.

It's pretty common to have reverse proxy in different machine - then it should bind to public interface.

Reply:
Yes, good clarification.